tests/builds_test.go

```diff
@@ -73,6 +73,7 @@ var _ = Describe("deis builds", func() {
 			})
 
 			Specify("that user can't create a new build of that app from a nonexistent image using `deis pull`", func() {
+				builds.Create(user, app)
```

Comment:
We already created the app with `apps.Create()` in the `BeforeEach()`, so this shouldn't be needed.